### PR TITLE
Modify documentation to reflect changes in params

### PR DESCRIPTION
**Pull request by cdrage**
This commit modified the documentation within atomic app to reflect the
addition of dashes to separate provider config related information.

### CODE REVIEW

File: docs/providers/kubernetes/overview.md [hunk 1]
```diff
@@ -28,22 +28,22 @@ section of the answers.conf file. An example is below:
...
 specify a location for a provider config file.
 
-One example of specifying a `providerconfig` is below:
+One example of specifying a `provider-config` is below:
 
 ```
 [general]
 provider: kubernetes
-providerconfig: /host/home/foo/.kube/config
+provider-config: /host/home/foo/.kube/config
```
Comment (dustymabe) [inline docs/providers/kubernetes/overview.md]: can we also get this updated to remove `/host/` from this ? separate commit would be fine.. I think there was a time where we didn't do the conversion so the user had to add it themselves

**Comment by dustymabe** [comment]
See the one comment.. Otherwise LGTM.

**Comment by rtnpro** [comment]
@cdrage You missed to update ``docs/providers/marathon/overview.md``.

**Comment by dustymabe** [comment]
> @cdrage You missed to update docs/providers/marathon/overview.md.

hmm did we also miss marathon in PR685? we really need automated testing for this.

**Comment by cdrage** [comment]
Updated with comments from @dustymabe and @rtnpro

**Comment by cdrage** [comment]
woops, wrong button, re-opened

**Comment by dustymabe** [comment]
LGTM @rtnpro ?

**Pull request merged**